new/LIT5992Miracle.xml

```diff
@@ -8,6 +8,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <title xml:lang="gez" xml:id="t1">ተአምረ፡ ኢየሱስ፡ ክርስቶስ፡</title>
                 <title xml:lang="gez" corresp="#t1" type="normalized">Taʾammǝra ʾIyasus Krǝstos</title>
                 <title xml:lang="en" corresp="#t1">Miracle of Jesus Christ</title>
+                <title xml:lang="en" xml:id="t2">Miracle of Jesus Christ:  A sick person is healed from demonic possession by reciting a prayer to Jesus Christ</title>
```

Comment:
Just to be perfect: "Miracle of Jesus" (without "Christ" as you entitled all the other miracles)